src/trans-word-level/instantiate_word_level.cpp

```diff
@@ -101,19 +123,14 @@ void wl_instantiatet::instantiate_rec(exprt &expr)
 {
   instantiate_rec(expr.type());
 
-  if(expr.id()==ID_next_symbol || expr.id()==ID_symbol)
+  if(expr.id() == ID_next_symbol)
   {
-    const irep_idt &identifier=expr.get(ID_identifier);
-
-    if(expr.id()==ID_next_symbol)
-    {
-      expr.id(ID_symbol);
-      expr.set(ID_identifier, timeframe_identifier(current+1, identifier));
-    }
-    else
-    {
-      expr.set(ID_identifier, timeframe_identifier(current, identifier));
-    }
+    expr.id(ID_symbol);
+    expr = timeframe_symbol(current + 1, to_symbol_expr(expr));
```

Comment:
See my comment above: given you want to rewrite `expr`, use `std::move` on the second argument.

Reply:
done